st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -44,14 +45,20 @@ def __init__(self):
         self._coordinator = coordination.get_coordinator()
         self.get_one_db_method = self.__get_by_name
 
-    @jsexpose(arg_types=[str])
-    def get_one(self, name):
+    @jsexpose(arg_types=[str, str])
+    def get_one(self, name, decrypt='false'):
```

Comment:
>I thought we already do this somewhere, but maybe we don't...

I checked we don't. 

>I don't think it should be bad, just a couple of lines of code in jsexpose..

That was my initial approach. I couldn't get it to work. I was complaining about it on slack. I can try again once PR is merged.